### PR TITLE
Bug/fix template content margin

### PR DESCRIPTION
**Pull request by michallaskowski**
Make sure these boxes are checked before submitting your Pull Request - thank you!

- [x] [SwiftLint](https://github.com/Realm/SwiftLint) is installed on your system
- [x] The project does not generate any warnings or errors
- [x] The tests are passing
- [x] Branch name is in line with [GitFlow](http://jeffkreeftmeijer.com/2010/why-arent-you-using-git-flow/)

I checked, and only `WebsiteAnalyticsWidget` had custom content margin. I made a snapshot test before changes, so the fix does not change widget's margins.

### CODE REVIEW

**Comment by michallaskowski** [comment]
Let's wait for #154 with this one

File: GrandCentralBoard/Widgets/WebsiteAnalytics/WebsiteAnalyticsWidget.swift [hunk 1]
```diff
@@ -48,7 +48,7 @@ final class WebsiteAnalyticsWidget: WidgetControlling {
     }
 
     private func updateWithReports(reports: [PageViewsRowReport]) {
-        let reportsSlice = reports[0...min(3, reports.count)]
+        let reportsSlice = reports.count > 0 ? reports[0..<min(3, reports.count)] : []
```
Comment (ncreated) [inline GrandCentralBoard/Widgets/WebsiteAnalytics/WebsiteAnalyticsWidget.swift]: You don't need ternary operator here. `0..<0` is just an [empty range](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/BasicOperators.html) and results in empty array.

```swift
let slicedReports = reports[0..<min(3, reports.count)]
```

**Comment by ncreated** [comment]
@michallaskowski just one comment from CR. Looks good 👌. Like you said, let's wait for #154 that adds more complexity in `WebsiteAnalyticsWidgetSnapshotTests.swift`.

File: GrandCentralBoard/Widgets/WebsiteAnalytics/WebsiteAnalyticsWidget.swift (hunk 1 above)
Comment (michallaskowski) [inline GrandCentralBoard/Widgets/WebsiteAnalytics/WebsiteAnalyticsWidget.swift]: Ok, thanks. I will test in a playground, didn't know that.

**Comment by krris** [comment]
Just to let you know https://github.com/macoscope/GrandCentralBoard/pull/154 is closed.

**Comment by krris** [comment]
Nice fix 🎆

**Pull request merged**